Makefile.am

```diff
@@ -29,11 +29,7 @@ AM_CXXFLAGS=
 AM_LDFLAGS =
 include $(top_srcdir)/coverage.mk
 
-SUBDIRS = dispatch xmlcommand ppt server http cmdln standalone docs bin templates hello_world
-
-if USE_AWS_SDK
-SUBDIRS += aws
-endif
+SUBDIRS = dispatch xmlcommand ppt server http cmdln aws standalone docs bin templates hello_world
```

Comment:
note: because aws is now required for standalone, i think we need to make it non-optional in the rest of the configuration. if this is *not* true, and there's a better approach, lmk---otherwise I'll make that non-optional change here.

- [ ] Get rid of USE_AWS_SDK all over, as aws is no longer optional